packages/rulesets/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Change Log - @microsoft.azure/openapi-validator-rulesets
 
-## 2.1.6
+This log was last generated on Fri, 08 Aug 2025 18:55:31 GMT and should not be manually modified.
```

Comment:
Please update this file manually (rather than using an update tool), and backfill the changelog for 2.1.7.